package.json

```diff
@@ -37,11 +37,15 @@
   },
   "homepage": "https://github.com/maxdome/dumpinator#readme",
   "dependencies": {
-    "eslint": "^3.10.2",
-    "eslint-config-airbnb-base": "^10.0.1",
-    "eslint-plugin-import": "^2.2.0"
+    "lodash": "^4.17.2",
+    "superagent": "^2.3.0"
   },
   "devDependencies": {
-    "mocha": "^3.1.2"
+    "eslint": "^3.10.2",
+    "eslint-config-airbnb-base": "^10.0.1",
```

Comment:
For new repos I would try to extend from the current version of "eslint-config-airbnb" and modify the needed rules.